src/comp_dna.py

```diff
@@ -1,15 +1,6 @@
 def complementary_dna(dna):
-    comp_dna = ""
-    for char in dna:
-        if char == "C":
-            comp_dna = comp_dna + "G"
-        elif char == "T":
-            comp_dna = comp_dna + "A"
-        elif char == "G":
-            comp_dna = comp_dna + "C"
-        elif char == "A":
-            comp_dna = comp_dna + "T"
-    return comp_dna
+    trans_map = {"A": "T", "T": "A", "C": "G", "G": "C"}
+    return "".join([trans_map[x] for x in dna])
```

Comment:
Checkout to main and pull the latest changes.  Then checkout back to this branch.  Then run "git merge main" to merge the changes from main into this branch.  That way this diff doesn't show up.

Reply:
okey dokey, thank you!

Reply:
Also once the merge has happened locally, the branch still needs to be pushed back up to GitHub